.gitignore

```diff
@@ -2,6 +2,7 @@
 /dist
 *.gql
 *.graphql
+/data
```

Comment:
This is a local setting for you. This path is only mentioned in one example comment.

I'd suggest adding a convention to your global gitignore like `*.local` and then this could be `.db.local`.
Or you could move it outside the project dir.

Reply:
I saw these settings in our gitignore:

.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json

which are also local.... nevertheless I have done this

Reply:
Yeah i go back and forth on whether these should be there. On one hand, yes they are a local thing and probably apply to every repo so they should be in your global gitignore. On the other hand, it's really standard to have these in the project's gitignore. Most repos have them, so it doesn't seem to be hurting anything.
In this case, `/data` could be anything, so that's what makes me want to push it exclusively to local. Also, this will be going away with edgedb migration, so that further's the thought of punting it instead of a better DX.

Reply:
Yeah if it's going away we don't need it here. Good call then